### DISCUSSION TITLE
Having problem on firefox browser

### DISCUSSION THREAD

**Discussion by MohammadTaseenKhan** [jeremyckahn/chitchatter#35] [Q&A] [answered]
@jeremyckahn 

> Recently I am facing a new problems like when i use Firefox browser i cant send any message, even cant receive any message even no notification coming to me

Sending Message:
![image](https://user-images.githubusercontent.com/85288261/194734929-db736a32-3483-46bf-b63a-c31c1dfcdf00.png)

Receiving message:
![image](https://user-images.githubusercontent.com/85288261/194734944-537abcdc-dbdd-4178-99bc-eadfceaf0840.png)

It doesn't even showing how many numbers of peers are connected

![image](https://user-images.githubusercontent.com/85288261/194734976-622ca92c-ac27-4ad0-8fb8-5c1684d5fff1.png)

same as well this also

![image](https://user-images.githubusercontent.com/85288261/194734993-259ef90a-eba0-47b9-afea-914f940820b9.png)

https://chitchatter.im/public/85ccdfc7-b992-411d-9d98-0a5c2781138e

**Comment by jeremyckahn** [comment, marked as answer]
Thank for you for reporting this @MohammadTaseenKhan! I have followed up in #36.
